crates/katana/executor/src/implementation/blockifier/mod.rs

```diff
@@ -23,6 +23,8 @@ use crate::{
     ExecutorFactory, ExecutorResult, ResultAndStates, SimulationFlag, StateProviderDb,
 };
 
+pub(crate) const LOG_TARGET: &str = "katana::executor::implementation::blockifier";
```

Comment:
```suggestion
pub(crate) const LOG_TARGET: &str = "katana::executor::blockifier";
```

can we shorten this, seems a bit too long